site/docs/v1/tech/core-concepts/entity-management.adoc

```diff
@@ -176,37 +176,40 @@ It is also not possible to rename or otherwise customize scopes used with Entity
-* `scim:user:delete` - link:/docs/v1/tech/apis/scim/SCIMUser#delete-a-user[Delete a User]
-* `scim:user:read` - link:/docs/v1/tech/apis/scim/SCIMUser#retrieve-a-user[Retrieve a User]
-* `scim:user:update` - link:/docs/v1/tech/apis/scim/SCIMUser#update-a-user[Update a User]
+However, FusionAuth ships with two SCIM default Entity Types that you can use.
+
+== [FusionAuth Default] SCIM client
+The default SCIM client Entity Type is the external SCIM client calling link:/docs/v1/tech/apis/scim/[SCIM API endpoints].
+
+== [FusionAuth Default] SCIM server
+The default SCIM server Entity Type is FusionAuth acting as the SCIM server and defines the available permissions for calling link:/docs/v1/tech/apis/scim/[SCIM API endpoints].
+
+image::core-concepts/entity-type-scim-add.png[Home Screen SCIM Entity Types,width=1200]
+
+=== SCIM server permissions
+Each permission enables or disables access to an API endpoint. These are required for any Entity Type you create which will be used to create Entities for SCIM authentication.
+
+When you create an Entity representing a SCIM server, you can choose which permissions it will expose by only creating those permissions that are needed by the clients. The permissions names must match the values in the table below. When you are creating an Entity representing a SCIM client, you must select one or more of the permissions available on the SCIM server entity in order for clients using those credentials to have access to the associated API. This allows you to control access to each individual API based on which permissions you decide to create in your SCIM server Entity Type, and then when you create a SCIM client Entity, you can choose which permissions to enabled for that specific Entity as well.
+
+Below is a list of available permissions that can be included in your SCIM server entity type. Each controls access to a single API endpoint. You can choose to create as few or as many of these as you require. The names must match exactly.
+
+.SCIM server permissions
+[cols="1,1"]
+|===
+| Name | API
+| `scim:user:create` | link:/docs/v1/tech/apis/scim/SCIMUser#create-a-user[Create a User]
+| `scim:user:read` | link:/docs/v1/tech/apis/scim/SCIMUser#retrieve-a-user[Retrieve a User]
+| `scim:user:update` | link:/docs/v1/tech/apis/scim/SCIMUser#update-a-user[Update a User]
+| `scim:user:delete` | link:/docs/v1/tech/apis/scim/SCIMUser#delete-a-user[Delete a User]
+| `scim:enterprise:user:create` | link:/docs/v1/tech/apis/scim/SCIMEnterpriseUser#create-an-enterpriseuser[Create an EnterpriseUser]
+| `scim:enterprise:user:read` | link:/docs/v1/tech/apis/scim/SCIMEnterpriseUser#retrieve-an-enterpriseuser[Retrieve an EnterpriseUser]
+| `scim:enterprise:user:update` | link:/docs/v1/tech/apis/scim/SCIMEnterpriseUser#update-an-enterpriseuser[Update an EnterpriseUser]
+| `scim:enterprise:user:delete` | link:/docs/v1/tech/apis/scim/SCIMEnterpriseUser#delete-an-enterpriseuser[Delete an EnterpriseUser]
```

Comment:
The text in the screen shot says `Enterprise User` instead of `EnterpriseUser`, these should probably match.